src/app/basic/basic-auto.component.ts

```diff
@@ -31,7 +31,9 @@ import { DataService } from '../data.service';
       >
       </ngx-datatable>
     </div>
-  `
+  `,
+  standalone: true,
```

Comment:
Not a finding just a general comment: Angular 19 would already have `standalone: true` as the default. So considering we're still on Angular 18, we introduce now an explicit `standalone: true`, just to remove it again with Angular 19, making the whole effort here somewhat verbose.

Reply:
I was also surprise when I saw that we are only on 18 here. But the migration to 19 should remove the flag.